source/deploy/postgres-migration.rst

```diff
@@ -193,6 +193,33 @@ Ensure that the ``mmuser`` user in PostgreSQL is the owner of the schema.
 
 Then, re-run the command from step 2.
 
+Incoming webhook channel names become case-sensitive after migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After migrating from MySQL to PostgreSQL, incoming webhooks that previously worked with mixed-case channel names (e.g., ``"Tech-Talk"``) will fail with the following error:
+
+.. code-block:: text
+
+  GetChannelByName: Channel does not exist.
+
+This occurs because Mattermost does not allow creating channels with uppercase names - all new channels must have lowercase names.
+Additionally, MySQL is case-insensitive when matching channel names, while PostgreSQL is case-sensitive. This means that webhook payloads that worked with MySQL's forgiving case-insensitive matching will fail with PostgreSQL's strict case-sensitive matching.
+In rare cases, older channels may still exist with mixed-case names in your database. These channels are considered invalid and may cause issues when performing operations like updating channel headers.
+
+**Solution:**
+
+Update all webhook payloads to use lowercase channel names.
+
+**Prevention:**
+
+Before migrating to PostgreSQL:
+
+1. Audit your incoming webhooks and update any that use mixed-case channel names in their payloads
+2. Check for any legacy channels with mixed-case names that may have been created in earlier versions
+3. Consider renaming any mixed-case channels to lowercase to prevent issues
```

Comment:
```suggestion
1. Audit your incoming webhooks and update any that use mixed-case channel names in their payloads.
2. Check for any legacy channels with mixed-case names that may have been created in earlier versions.
3. Consider renaming any mixed-case channels to lowercase to prevent issues.
```